### DISCUSSION TITLE
[mpvnet] 非全屏状态 uosc界面元素操控异常

### DISCUSSION THREAD

**Discussion by yubanmeiqin9048** [hooke007/mpv_PlayKit#256] [libmpv]
非全屏时，唤出菜单后点不动，全屏后可以
![)LZ_PAU2$Y~% @S5T0OK{M](https://user-images.githubusercontent.com/64676973/213731191-59826b6b-e104-4021-80d5-a16ba3c3896e.png)

**Comment by hooke007** [comment]
不是我的问题，上游4.4.0还是正常的，4.5.0的窗口模式下的操作失效了，向上游报告吧

**Comment by Lonely233233** [comment]
![image](https://user-images.githubusercontent.com/106970209/229351279-6d783a13-0a6f-4e9d-997e-4a713ca1f25b.png)
这下面的条状能去掉吗

**Reply by hooke007** [reply]
看脚本选项
